elifetools/tests/test_xmlio.py

```diff
@@ -1,11 +1,6 @@
 from __future__ import absolute_import
 
-try:
-    # Python 2
-    from StringIO import StringIO
-except ImportError:
-    # Python 3
-    from io import StringIO
+from io import StringIO
```

Comment:
You switched it back to StringIO but without the Python 2 import. This is can be Python 3 only now or did you do something else that would make the import work for Python 2? (BTW there is also `six.StringIO` if that made anything easier)

Reply:
I was reviewing the Python 2 documentation and it was showing ``io.StringIO`` has been available since 2.6 (https://docs.python.org/2/library/io.html?highlight=io#io.StringIO). I gave it a try, and in this case it just needed some tweaks because it would only accept unicode as the input, not str. In Python 3 it seems to accept it too.

It seems to work in this limited usage in one set of test scenarios. I'm not sure if using ``io.StringIO`` for all uses will result in the right output, regarding the character encoding, but it could be something we could try in other libraries.

Reply:
Thank you, I did not know that. (using `six` probably wouldn't help here)